### ISSUE TITLE
GridView disappears from drag contener during dragging

### ISSUE THREAD

**Issue by piotrwojtowicz** [h6ah4i/android-advancedrecyclerview#319]
I've noticed today that during dragging GridView disappears from drag contener (see attached images). It's not big deal, maybe it even looks better :) Tested on emulator API 10, 18, 23 and on device API 23. 

![s1](https://cloud.githubusercontent.com/assets/11367760/20093609/c94b824c-a59d-11e6-8e87-ae577e5ab414.png)

![s2](https://cloud.githubusercontent.com/assets/11367760/20093608/c9480c52-a59d-11e6-93cc-833754a8413d.png)

**Comment by h6ah4i** [comment]
@piotrwojtowicz Hi. Thanks for reporting the issue. Could you upload sample project (or code) which can reproduce the issue? 

---

**FYI**

While dragging, this library uses pre-rendered view image for dragging item. It seems something wrong when creating a pre-rendered bitmap image. If possible, try to debug the following method;

https://github.com/h6ah4i/android-advancedrecyclerview/blob/master/library/src/main/java/com/h6ah4i/android/widget/advrecyclerview/draggable/DraggingItemDecorator.java#L411

**Comment by piotrwojtowicz** [comment]
Sure, source attached, it's your example changed, in file MinimalDraggableExampleActivity.java I added simple GridView. Few things can be noticed in "Draggable (minimal)" example: 

- when activity appears we shortly see scroll bars next to each grid view (maybe because of fixed list item height) - that not too much important :)
- when dragging list element grid view disappears and appears when list element is dropped

[src.zip](https://github.com/h6ah4i/android-advancedrecyclerview/files/578381/src.zip)

**Comment by h6ah4i** [comment]
@piotrwojtowicz Could you try the [`fix_issue_319`](https://github.com/h6ah4i/android-advancedrecyclerview/tree/fix_issue_319) branch code via JitPack? Maybe the issue has been fixed.

#### Gradle:

```gradle
repositories {
    maven { url "https://jitpack.io" }
}

dependencies {
    compile 'com.github.h6ah4i:android-advancedrecyclerview:c215fa50032ad0fad853ba8b5407a293a59fbb41'
}
```

**Comment by piotrwojtowicz** [comment]
It's working perfect now :) Thank you.

**Comment by h6ah4i** [comment]
This issue has been merged and released as v0.10.1 🎉
